lib/voyage/templates/api_base_controller.rb

```diff
@@ -0,0 +1,38 @@
+module Api
+  class BaseApiController < ApplicationController
+    # Disable CSRF protection for API calls
+    protect_from_forgery with: :null_session
```

Comment:
Using :null_session here overrides the CSRF protection that we inherit from the ApplicationController.